components/Tabs/src/private/MDCItemBarStyle.h

```diff
@@ -65,6 +65,9 @@
 /** Padding in points between the title and image components, according to the MD spec. */
 @property(nonatomic) CGFloat titleImagePadding;
 
+/** Maximum number of lines of text allowed. */
+@property(nonatomic, assign) NSInteger titleNumberOfLines;
```

Comment:
If we are always supposed to support wrapping for text-only items, we could eliminate this property.
If this only applies to text-only tabs, please indicate that in the comment / name.
textOnlyNumberOfLines perhaps?

Reply:
If there's an image, I believe we limit to a single line of text. In that case the spec isn't explicit, but I'm pretty sure the intention is a single line of text once images are present.  Two lines of text are also not permitted for Tabs on the bottom (Botton Nav).

The naming gets wonky, since it's "the maximum possible number of lines of title text depending on other factors."  How about "textOnlyMaximumNumberOfLines" ?

Reply:
Got it.
Consider: textOnlyNumberOfLines as it would match UIKit / UILabel.

Reply:
Done.